common/djangoapps/student/tests/test_scope_resolver.py

```diff
@@ -69,14 +140,16 @@ def test_email_resolver(self):
         resolver = StudentEmailScopeResolver()
 
         emails_resultset = resolver.resolve(
-            'student_email_resolver',
+            'user_email_resolver',
             {
                 'user_id': test_user_1.id,
             },
             None
         )
 
-        self.assertTrue(test_user_1.email in emails_resultset)
+        emails = [email['email'] for email in emails_resultset]
```

Comment:
This list comprehension is a bit difficult to comprehend, with all the "email" names going on